src/set_theory/game/pgame.lean

```diff
@@ -660,6 +660,107 @@ def relabel_relabelling {x : pgame} {xl' xr'} (el : x.left_moves ≃ xl') (er :
   relabelling x (relabel el er) :=
 relabelling.mk el er (λ i, by simp) (λ j, by simp)
 
+/-- `is_relabelled_option x y` means that `x` is either a either a left or right option of `y`, up
+to relabelling. Contrast with `is_option`. -/
+@[mk_iff is_relabelled_option_iff_left] inductive is_relabelled_option : pgame → pgame → Prop
+| relabelling_left {x₁ x₂ y : pgame} (h : is_option x₁ y) (e : x₁.relabelling x₂) :
+    is_relabelled_option x₂ y
+
+theorem is_option.is_relabelled_option {x y : pgame} (h : is_option x y) :
+  is_relabelled_option x y :=
+⟨h, relabelling.refl x⟩
+
+theorem is_relabelled_option.move_left {x : pgame} (i : x.left_moves) :
+  is_relabelled_option (x.move_left i) x :=
+(is_option.move_left i).is_relabelled_option
+
+theorem is_relabelled_option.move_right {x : pgame} (j : x.right_moves) :
+  is_relabelled_option (x.move_right j) x :=
+(is_option.move_right j).is_relabelled_option
+
+theorem is_relabelled_option.mk_left {xl xr : Type u} (xL : xl → pgame) (xR : xr → pgame)
+  (i : xl) : is_relabelled_option (xL i) (mk xl xr xL xR) :=
+@is_relabelled_option.move_left (mk _ _ _ _) i
+
+theorem is_relabelled_option.mk_right {xl xr : Type u} (xL : xl → pgame) (xR : xr → pgame)
+  (j : xr) : is_relabelled_option (xR j) (mk xl xr xL xR) :=
+@is_relabelled_option.move_right (mk _ _ _ _) j
+
+theorem is_relabelled_option.congr_left {x₁ x₂ y : pgame} (e : x₁.relabelling x₂) :
```

Comment:
Since you don't use this below, it probably makes sense to put it next to `congr_right`